packages/components/src/styles/footnote/footnote.css

```diff
@@ -0,0 +1,16 @@
+.sd-footnote {
+  &--reset {
+    counter-reset: footnote;
+  }
+
+  @apply text-neutral-700 text-sm flex gap-2;
```

Comment:
`flex` and `gap-2` are only needed for number, right?

I would move it there, to make that more clear.

Reply:
Agreed